project_euler/problem_26/sol1.py

```diff
@@ -35,8 +53,22 @@ def find_digit(numerator: int, digit: int) -> int:
     return the_digit
 
 
-# Tests
-if __name__ == "__main__":
-    import doctest
+def solution(n: int = 1000) -> int:
```

Comment:
Instead of making another function, change the name of the original function and add default arguments to it.
Also, please don't remove the doctest part in `if __name__ == "__main__":` block

Reply:
Thanks for the review! I thought that the `solution()` function needed to only have one argument. But after looking at the test script more I see that it just needs to have default arguments. 

I have made the changes you requested. 

I was a little unsure how to to modify the `print(solution(int(str(input()).strip())))` section in the `if __name__ == "__main__":` block to deal with multiple arguments. So I tried to make it as similar to the problem description as I could. Let me know if that's okay.

Thanks.